packages/moonstone/TooltipDecorator/util.js

```diff
@@ -180,7 +181,17 @@ const getPosition = function (tooltipNode, clientNode, arrowAnchor, tooltipDirec
  */
 const getArrowPosition = function (tooltipNode, clientNode, overflow, rtl) {
 	if (overflow.isOverWide) {
-		return rtl ? 1 - ((window.innerWidth - clientNode.right + (clientNode.width / 2)) / tooltipNode.width) : (clientNode.left + (clientNode.width / 2)) / tooltipNode.width;
+		const tooltipWidth = tooltipNode.width;
+		// finding out where the middle of the clientNode is and figuring out where that is in relation to the tooltip node in percentage between 0 and 1
+		const arrowPosition = rtl ? 1 - ((window.innerWidth - clientNode.right + (clientNode.width / 2)) / tooltipWidth) : (clientNode.left + (clientNode.width / 2)) / tooltipWidth;
+
+		const borderRadius = 33; // based on @moon-tooltip-border-radius
```

Comment:
This value needs to be RI converted so it represents the correct amount of pixels on 720p and 4k: `ri.scale(33)`